Documentation/ceph-encryption.md

```diff
@@ -0,0 +1,225 @@
+#  Ceph encryption at rest
+
+## Dmcrypt
+Dm-crypt is a Linux Kernel submodule that provides encryption support. It allows
+users to maps a virtual device to a physical device, that way the data can be
+encrypted on disk but decrypted when read from the virtual device.
+
+![](encryption-dataflow.png)
```

Comment:
Looks like I missed including this diagram. Will do next iteration. It is nothing important, just abstraction of how dmcrypt works.